src/components/PokemonList.js

```diff
@@ -52,30 +61,40 @@ class PokemonList extends Component {
         localStorage.setItem(this.pageKey, newOffset/limit+1);
     }
 
+    toggleFavOnly() {
+        const newFav = !this.state.favOnly;
+
+        this.setState({
+            favOnly: newFav,
+        })
+    }
+
     render() {
         const {
+            favOnly,
             offset,
             limit,
             count,
             pokemons,
+            favPokemons,
         } = this.state;
         return (
             <div>
-                <h1>Pokemons</h1> 
+                <h1>Pokemons</h1>
+                <button onClick={this.toggleFavOnly}>Only favorite pokemons</button>
```

Comment:
Please change a text of the button appropriately. When you show only favorite pokemons you should show a button with "Show all"